### PR TITLE
Add lucide icons link

### LINKED ISSUE #1822: [Add] lucide.dev to icons in frontend
### Site URL

https://lucide.dev/

### Description of the above provided site

A lightweight icons made by their community, customizable and available on a huge amount of frameworks and copyable as SVGs and able to be pre-configured.

### Category

Frontend

### Sub Category

icons

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [X] I am participating in Hacktoberfest 2023

### PR DESCRIPTION
**Pull request by itaim18**
closes #1822 

## Changes proposed

Added the link as another object in `database/frontend/icons.json`

## Screenshots

image

### CODE REVIEW

**Comment by CBID2** [comment]
Hi @itaim18. Welcome to LinksHub! :) We don't accept spam pull requests, so I'm putting yours in draft mode so I can review your issue. In the future, please wait until your issue has been approved and assigned to you before you submit a pull request. Refer to our [Contributing Guide](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md) to learn how to effectively contribute to the project. Happy Contributing! 😊

**Comment by itaim18** [comment]
Oh sure!😅
Yet, it's a good resource which I have used in way too many projects to avoid it for your dictionary.
Definitely right about that approval protocol :)

File: database/frontend/icons.json [hunk 1]
```diff
@@ -220,7 +220,14 @@
     "name": "Iconoir",
     "description": "A high-quality selection of free icons. Has over 1,368 icons ready to use for your projects.",
     "url": "https://iconoir.com/",
-    "category": "fontend",
+    "category": "frontend",
+    "subcategory": "icons"
+  },
+  {
+    "name": "Lucide",
+    "description": "Lucide is a Beautiful & consistent icon toolkit made by the community.",
```
Comment (CBID2) [inline database/frontend/icons.json:228]: [proofreading] Interesting resource! :)  However, when you're describing your resources, avoid doing things like capitalizing words that are not at the beginning of a sentence (programming languages are an exception to this). It makes descriptions unclear.  
```suggestion
    "description": "Lucide is a beautiful & consistent icon toolkit made by the community.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @itaim18! :)  I pointed out a proofreading error in my inline comment. In the future, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Other than that, you're off to a great start! :)

**Comment by CBID2** [comment]
> Oh sure!😅 Yet, it's a good resource which I have used in way too many projects to avoid it for your dictionary. Definitely right about that approval protocol :)

Great @itaim18! :) Tag me, @Anmol-Baranwal, or @rupali-codes when you're ready

**Comment by itaim18** [comment]
On it!

**Comment by itaim18** [comment]
@CBID2 Ready!

**Review by CBID2 — APPROVED** [review]
Well done @itaim18! :) @Anmol-Baranwal, it's your turn! :)

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

**Pull request merged**